src/libpriv/05-rpmostree.install

```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/bash
+# Check if install.conf is missing or does not include layout=ostree
+if [ ! -f /usr/lib/kernel/install.conf ] || ! grep -q layout=ostree /usr/lib/kernel/install.conf; then
+    echo "install.conf not found or does not include layout=ostree, not calling into rpm-ostree kernel-install."
```

Comment:
I think we can probably skip this message as it will be distracting for dnf or other system users.